DEV-9/DEV-9/Tests/MailLoginTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using OpenQA.Selenium.Chrome;
+
+namespace DEV_9.Tests
+{
+ 
+    public class LoginTestsMailRu
+    {
+
+        IWebDriver Driver { get; set; }
+        WebDriverWait Wait { get; set; }
+
+
+        [SetUp]
+        public void StartBrowser()
+        {
+            this.Driver = new ChromeDriver("C:/Users/User/Downloads");
+            this.Wait = new WebDriverWait(Driver, TimeSpan.FromMinutes(1));
+        }
+
+
+        [Test]
+        public void Login_With_Correct_Data()
+        {
+            var loginPage = new Page_Objects.MailLoginPage (Driver);
+            var mainPage = loginPage.LoginAs("demitrodub", "qwaqwaqwa11");
+            Assert.True(mainPage.FindWriteLetterButton().Displayed);
+        }
+
+        [Test]
+        public void Login_EmptyInput_ErrorMessage()
+        {
+            var loginPage = new Page_Objects.MailLoginPage(Driver);
+            loginPage.LoginAs(string.Empty, string.Empty); 
+            //Wait.Until(t => Driver.FindElements(By.XPath("//input[contains(@class, 'is-error']")).Any());
+            var log = Driver.FindElements(By.XPath("//input[contains(@class, 'is-error']"));
```

Comment:
do not mix different layers. it is needed to operate only with PO and helper methods in your tests